bootstrap/terraform/function/function-project/vlt-auth-func-test/__init__.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import requests
+import os
+
+import azure.functions as func
+
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    # return get_sys_assigned_id_token()
+    return get_user_assigned_id_token()

```

Comment:
Nice! Looks like this works for both system and user assigned.

Reply:
yep, I tested with both